buildSrc/src/main/groovy/org/logstash/gradle/tooling/ListProjectDependencies.groovy

```diff
@@ -0,0 +1,50 @@
+package org.logstash.gradle.tooling
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.Input
+import org.gradle.api.artifacts.Dependency
+
+import java.nio.file.Files
+
+class ListProjectDependencies extends DefaultTask {
+
+    @Input
+    String report = 'licenses.csv' //project.file("${project.buildDir}/reports/dependency-license/licenses.csv")
+
+    @OutputFile
+    File reportFile = project.file("${project.buildDir}/reports/dependency-license/" + report)
+
+    @OutputDirectory
+    File reportDir = reportFile.toPath().parent.toFile()
+
+    ListProjectDependencies() {
+        description = "Lists the projects dependencies in a CSV file"
+        group = "org.logstash.tooling"
```

Comment:
Is it relevant that this group is different to the package name?

Reply:
I've a doubt here if use a group name like `Logstash tooling` or use the punctuacted version.
It's just a string so don't use the package name is not a problem, my objective is to identify the Logstash's tooling tasks.
Which name to you think fit best?

Reply:
What you've got is fine, I think